product-service/db_creation_script.sql

```diff
@@ -0,0 +1,33 @@
+--create extension if not exists "uuid-ossp";

```

Comment:
do we need this file?

Reply:
Quoting from Task 4.1:
> Write SQL script to fill tables with test examples. Store it in your GIT repository. Execute it for your DB to fill data.

Reply:
My point is that ideally this script should be executable at any time, to re-seed your DB. Nevermind anyway